### DISCUSSION TITLE
5.2.1 Hotfix Release Date

### DISCUSSION THREAD

**Discussion by SMitch5000** [dotnet/SqlClient#2530] [General]
Hello,

Any Idea when hotfix 5.2.1 will be released?

**Comment by JRahnama** [comment]
quiet soon. Date is yet to be confirmed, but it should not take long.

**Comment by JRahnama** [comment]
To add more after today's discussion, we are planning to have the release published by the end of this month.

**Comment by paulmedynski** [comment]
[5.2.1](https://github.com/dotnet/SqlClient/releases/tag/v5.2.1) was released on 31-May-2024.
